src/routes/Home/Home.tsx

```diff
@@ -68,7 +99,18 @@ const Home = () => {
         <div className="col-span-3 print:col-span-3 flex flex-col gap-8">
           <div>
             <h2 className="text-heading-2-mobile sm:text-heading-2 mb-5">Uudet sisällöt</h2>
-            <p className="bg-todo h-[328px] flex items-center justify-center rounded">TODO</p>
+            <div className="flex flex-row gap-6">
+              {newContent.map((c) => (
+                <MediaCard
+                  key={c.id}
+                  label={c.title}
+                  description={c.description}
+                  imageSrc={c.imageSrc}
+                  imageAlt={c.imageAlt}
+                  tags={c.tags}
+                />
+              ))}
+            </div>
```

Comment:
It is not yet responsive. Was it intended to be done later?

Reply:
I guess so. In the design the cards are supposed to be in a carousel 
![image](https://github.com/user-attachments/assets/44d326ca-170b-4b7c-b7a7-109ee0aab656)